lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/vehicle/sensor/LidarData.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.lib.objects.vehicle.sensor;
+
+import org.eclipse.mosaic.lib.util.gson.PolymorphismTypeAdapterFactory;
+
+import com.google.gson.annotations.JsonAdapter;
+
+import java.io.Serializable;
+
+public class LidarData implements Serializable {
+    @JsonAdapter(PolymorphismTypeAdapterFactory.class)
+    private final Object lidarData;
```

Comment:
Why is this an Object and why is it called `lidarData` inside of the `LidarData`-class? This adds unnecessary confusion.

Do we not have a class representing the point clouds instead of using a generic object?